english/named_entity_recognition.md

```diff
@@ -17,6 +17,7 @@ corpus tagged with four different entity types (PER, LOC, ORG, MISC). Models are
 
 | Model           | F1  |  Paper / Source | Code |
 | ------------- | :-----:| --- | --- |
+| InferNER (Moemmur et al. 2021) | 93.76| [InferNER: an attentive model leveraging the sentence-level information for Named Entity Recognition in Microblogs](https://journals.flvc.org/FLAIRS/article/view/128538) | |
```

Comment:
This should be ordered based on the F1 score, with the best model at the top. Could you add your result to the second row, please?

Reply:
Thank you, 
Please check the latest commit.

Reply:
Thanks. Looks good now.